documentation/release_notes.markdown

```diff
@@ -4,87 +4,271 @@ title: "Puppet Server: Release Notes"
 canonical: "/puppetserver/latest/release_notes.html"
 ---
 
+## Puppet Server 1.1.0
```

Comment:
Huh, I hadn't thought about renaming the file. That will cause problems, like everything does :) but maybe it'll solve more than it causes. 

Mainly, we expect the release notes for a project to have a predictable URL for linking. We link from two kinds of places:

* The sidebar nav. Nice and contained, I guess easy enough to update when we cut a new release. And actually, if we just add a predictable version number in the same place every time, we could partially automate that.
* Random pages who knows where. So far I'm actually just seeing index.md in y'all's folder, but we might naïvely make more someday? We can try to keep from doing that.

The other option would be to use the docs website's ability to insert snippets into a page. That way, we'd keep release_notes.md, but its content would just be something like `{% partial ./release_notes_2.1.md %}`.

Uh, I guess long story short I'm :+1: on doing that, but let's talk for another minute about how to arrange it.